src/test/java/edu/ksu/canvas/impl/SubmissionImplTest.java

```diff
@@ -0,0 +1,58 @@
+package edu.ksu.canvas.impl;
+
+import edu.ksu.canvas.CanvasTestBase;
+import edu.ksu.canvas.interfaces.SubmissionWriter;
+import edu.ksu.canvas.model.assignment.Progress;
+import edu.ksu.canvas.requestOptions.SubmissionOptions;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class SubmissionImplTest extends CanvasTestBase {
+    private static final Logger LOG = Logger.getLogger(SubmissionImplTest.class);
+    public static final int COURSE_ID = 37836;
+    public static final int SECTION_ID = 41478;
+    public static final int ASSIGNEMNET_ID = 409522;
+    public static final String SOME_COMMENT_1 = "SOME COMMENT 1";
+    public static final String GRADE_1 = "5";
+    public static final String SOME_COMMENT_2 = "COMMENT 2";
+    public static final String GRADE_2 = "5.3";
+    public static final int STUDENT_ID_1 = 52359;
+    public static final int STUDENT_ID_2 = 90605;
+
+    SubmissionWriter submissionWriter;
+
+    @Before
+    public void setup() {
+        String url =  baseUrl  + "/api/v1/sections/" + SECTION_ID + "/assignments/" + ASSIGNEMNET_ID + "/submissions/update_grades";
+        LOG.info(url);
```

Comment:
Can this log statement contain a message, changed to DEBUG level, or be removed?